src/data/utils.spec.js

```diff
@@ -22,3 +23,18 @@ describe('parseAPITime', () => {
     expect(parseAPITime(formatAPITime(d)).valueOf()).toBe(day.valueOf());
   });
 });
+
+describe('correlationMetrics', () => {
+  test('Deaths vs. CLI correlations should equal what was published in the blog post', async () => {
+    let expected_metrics = {
+      r2At0: 0.43,
+      lagAtMaxR2: 20,
+      r2AtMaxR2: 0.81,
+    };
+    let actual_metrics = generateCorrelationMetrics(deathsTestData, cliTestData);
+    expect(actual_metrics.r2At0).toEqual(expected_metrics.r2At0);
```

Comment:
an alternative method to defining the values directly is to use snapshot testing: https://jestjs.io/docs/snapshot-testing

Reply:
That's a pretty neat feature.  Here's however, I wanted to be a little looser about the assertion, specifically in lags array that is returned.  Currently it's not sorted with respect to lag number, so if the ordering changes in the future, I didn't want to break the test.